web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -202,9 +259,69 @@ export default defineComponent({
                 mdi-arrow-right-circle
               </v-icon>
             </v-btn>
+            <v-menu
+              offset-x
+            >
+              <template #activator="{ on }">
+                <v-btn
+                  text
+                  icon
+                  class="ml-1"
+                  v-on="on"
+                >
+                  <v-icon>
+                    mdi-dots-vertical
+                  </v-icon>
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item
+                  v-for="(entry, i) in OverFlowMenuItems"
+                  :key="i"
+                  @click="() => handleOverflowMenu(item,entry.title)"
+                >
+                  <v-list-item-title>{{ entry.title }}</v-list-item-title>
+                </v-list-item>
+              </v-list>
+            </v-menu>
           </template>
         </v-data-table>
       </v-card>
     </v-card>
+    <v-row justify="center">
+      <v-dialog
+        v-model="deleteDialog"
+        :width="630"
+        class="ma-5"
+      >
+        <v-card>
+          <v-spacer />
+          <v-card-title class="ma-3 text-h4">
+            Are you sure you want to delete this submission in progress?
+          </v-card-title>
+          <v-card-text class="ma-3 text-h5">
```

Comment:
Can probably remove both of these classes.

Reply:
Removed